pkg/service/service.go

```diff
@@ -20,24 +20,33 @@
 package service
 
 import (
+	goflag "flag"
 	"fmt"
 	"strconv"
+
+	flag "github.com/spf13/pflag"
 )
 
-type Service struct {
+type Service interface {
+	Run()
+	InitOptions()
+}
+
+type ServiceListen struct {
 	Name string
 	Host string
```

Comment:
If the idea is to simplify/standardize nomenclature, I think this should be `BindAddress` not `Host`.